test/e2e/identityvalidator/identityvalidator.go

```diff
@@ -26,6 +27,10 @@ var (
 	keyvaultSecretVersion = pflag.String("keyvault-secret-version", "", "the version of the keyvault secret we are extracting with pod identity")
 )
 
+const (
+	contextTimeout = 150 * time.Second
```

Comment:
Is this supposed to be 120s or 150s?

Reply:
Keeping it at 150s to accommodate any VMSS identity assignment delays.